k8s/Makefile

```diff
@@ -39,6 +39,14 @@ export CELERY_WORKER_MEMORY_REQUEST ?= 128Mi
 export CELERY_WORKER_CPU_LIMIT ?= 2
 export CELERY_WORKER_MEMORY_LIMIT ?= 256Mi
 
+export CELERY_SCHEDULER_NAME ?= celery-beat
+export CELERY_SCHEDULER_REPLICA ?= 1
+export CELERY_SCHEDULER_CONCURRENCY ?= 1
+export CELERY_SCHEDULER_CPU_REQUEST 100m
```

Comment:
Does this need `?=` before `100m`?

Reply:
Good catch

Reply:
Fixed in a4c7d02f6ec2119197cd5267ddd3f23f91264cf3